packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -63,27 +63,48 @@ const Container = styled.header`
   }
   @media screen and (${designSystem.dimensions.viewports.desktop}) {
     grid:
-      [row1-start] 'header-content header-blank' ${designSystem.dimensions
+      [row1-start] 'header-top-menu header-searchbox' ${designSystem.dimensions
         .heights.header} [row1-end]
       / ${designSystem.dimensions.widths.pageContentWithMargins}
       minmax(${designSystem.dimensions.widths.pageNavigation}, 1fr);
   }
+
+  ${(props) =>
+    props.allowWideContentLayout
+      ? `@media screen and (${designSystem.dimensions.viewports.largeDesktop}) {
```

Comment:
I've implemented now the viewport for the wide layout. Problem is i'm not sure why the index nav isn't moving with the page size anymore when I make the window smaller. I guess this has something to do with my grid config here which probably has to be adjusted. Suggestions are welcome :)